src/include/migraphx/op/quantizelinear.hpp

```diff
@@ -38,6 +38,10 @@ namespace op {
 struct quantizelinear
 {
     std::string name() const { return "quantizelinear"; }
+
+    value attributes() const { return {{"pointwise", true}, {"point_op", 
+        "${function:max}(${function:min}(${function:round}(${function:convert}<float>(${0}) / ${1}) + ${function:convert}<float>(${2}), 127.0), -128.0)"}}; }
```

Comment:
I cannot think of a way to get rid of the hard coded constants for clip: -128 and 127. I tried `std::numeric_limits<decltype(${2})>::max()` or `...::lowest()` but that require the existence of ${2} in the first place.

Reply:
Yea this is getting kind of messy. Thinking about it more, we could just set `pointwise` to true and then run `rewrite_quantization` in `generate_pointwise`(we already run some passes before generating).

Reply:
Also, I'm assuming you prefer the same for both quantizelinear and dequantizelinear? Reason I'm asking is due to math in quantizelinear is much more straightforward and there's no harm to keep a definition there.

Reply:
> You mean to add it here? 

Yes.

> Also, I'm assuming you prefer the same for both quantizelinear and dequantizelinear? 

Yes.